grafana/version.txt

```diff
@@ -1 +1 @@
-7.0.1
+7.0.1-r1
```

Comment:
Looks great, just wondering what we are doing in the other images for the modifier here? 

I remember chatting with @vishwas-astro recently and wondered if we could use `+astro.1` as the modifier to make it clear across the board whats happening.

Reply:
Incrementing the number as we go of course.

Reply:
I think I tried that once and it failed in CI because the ci system checks for semver against pep 440. Here are examples that are compliant. https://www.python.org/dev/peps/pep-0440/#examples-of-compliant-version-schemes

it looks like we could maybe to `.astro1` Of course we could always change the semver check too. I'm game for whatever, I just found that -r1 worked. :)

Reply:
Ah, ok `.astro1` works for me. Then maybe we can start standardizing on that format across the rest of the vendors when we get to it. Mostly concerned with just getting something consistent that we can roll with.

Reply:
Version has been updated.